clap_derive/src/utils/doc_comments.rs

```diff
@@ -78,14 +78,14 @@ fn split_paragraphs(lines: &[&str]) -> Vec<String> {
         let len = slice
             .iter()
             .position(|s| is_blank(s))
-            .unwrap_or_else(|| slice.len());
+            .unwrap_or(slice.len());
 
         last_line += start + len;
 
-        if len != 0 {
-            Some(merge_lines(&slice[..len]))
-        } else {
+        if len == 0 {
             None
+        } else {
+            Some(merge_lines(&slice[..len]))
         }
```

Comment:
Why did clippy want this changed?

Reply:
Clippy weights an equality check as a simpler operation than a non-equality check

Reply:
reverted